datalad/distribution/update.py

```diff
@@ -15,23 +15,33 @@
 
 import logging
 from os.path import lexists, join as opj
+import itertools
 
 from datalad.interface.base import Interface
 from datalad.interface.utils import eval_results
 from datalad.interface.base import build_doc
 from datalad.interface.results import get_status_dict
-from datalad.support.constraints import EnsureStr
-from datalad.support.constraints import EnsureNone
+from datalad.support.constraints import (
+    EnsureStr,
+    EnsureNone,
+)
 from datalad.support.annexrepo import AnnexRepo
 from datalad.support.param import Parameter
-from datalad.interface.annotate_paths import AnnotatePaths
-from datalad.interface.common_opts import recursion_flag
-from datalad.interface.common_opts import recursion_limit
+from datalad.interface.common_opts import (
+    recursion_flag,
+    recursion_limit,
+)
 from datalad.distribution.dataset import require_dataset
 
-from .dataset import Dataset
-from .dataset import EnsureDataset
-from .dataset import datasetmethod
+from .dataset import (
+    Dataset,
+    EnsureDataset,
+    datasetmethod,
+)
+
+# needed API commands
+import datalad.distribution.subdatasets
+import datalad.core.local.save
```

Comment:
Hmm, I added it (this one, or one in another PR), because it did not find a method. Maybe I interpreted it wrong. Anyways, thanks for bringing this to my attention.